playbooks/support/responding.md

```diff
@@ -15,48 +15,67 @@ respond to these issues.
 ## Step 2: Raise
 
 1. When an incident is raised in the channel or identified in another way, the on-call team first **tracks the
-   incident** by creating a ticket in [Jira Ops](https://artsyproduct.atlassian.net/projects/INCIDENT/incidents),
-   based on the information they know at that time. Some workflow steps:
-
-   - In the description on the ticket, link to the slack thread in #incidents or #alerts.
+   incident** by creating a ticket in [OpsGenie](https://artsy.app.opsgenie.com/incident/list), based on the
+   information they know at that time. Some workflow steps:
+
+   - Select Artsy as the Impacted Service.
+   - Enter a quick description of the problem as the Incident message.
+   - Enter a link to the #incidents thread as the Incident description.
+   - Select a Priority level from the list using your gut instinct. Refer to the
+     [section on Severity of Incidents for guidance](#severity-of-incidents).
    - Take your best guess at the other information, it can always be edited later.
 
-2. Using the slack integration on the Jira Ops ticket, create a new slack channel to discuss resolution.
+2. Create a Slack channel for this incident for discussion of the technical resolution.
+
+   - Name it #incident-N and invite the other on-call engineer and any other active responders.
```

Comment:
Should we describe what `N` represents or do you think it's intuitive enough?

Reply:
I can add some clarification.